metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/Metrics.java

```diff
@@ -28,89 +28,100 @@
     AppPrefix("metacat"),
 
     /**
+     * Measure types
+     */
+    Count("count"),
+    Gauge("gauge"),
+    Timer("timer"),
+
+    /**
      * evnets.
      */
-    CounterEventAsync(AppPrefix + ".events.count.Async"),
-    CounterEventSync(AppPrefix + ".events.count.Sync"),
+    Events("events"),
+    CounterEventAsync(Name(Events, Count , "async")),
+    CounterEventSync(Name(Events, Count, "sync")),
 
     /**
      * thrift request.
      */
-    CounterThrift(AppPrefix + ".thrift.count.request"),
+    Thrift("thrift"),
+    CounterThrift(Name(Thrift, Count,   "request")),
 
     /**
      * hive sql lock error.
      */
-    CounterHiveSqlLockError(AppPrefix + ".count.hiveSqlLockError"),
-    CounterHiveExperimentGetTablePartitionsFailure(AppPrefix + ".hive.count.experimentGetPartitionsFailure"),
+    Hive("hive"),
```

Comment:
I think that Hive metrics should be kept within the Hive connector module. They have nothing to do with the main metacat system.